### ISSUE TITLE
Installing maven on 8-jdk downgrades java version

### ISSUE THREAD

**Issue by frangarcia** [docker-library/openjdk#110]
I am using the image openjdk:8-jdk and installing maven after running the container. Doing that, downgrades the java version. Those are the steps to reproduce the issue:
```
docker pull openjdk:8-jdk  
docker run -it openjdk:8-jdk

#inside the container
java -version
# output
# openjdk version "1.8.0_121"
# OpenJDK Runtime Environment (build 1.8.0_121-8u121-b13-1~bpo8+1-b13)
# OpenJDK 64-Bit Server VM (build 25.121-b13, mixed mode)

apt-get update && apt-get -y install maven

# after finishing the maven installation
java -version
# output
# java version "1.7.0_121"
# OpenJDK Runtime Environment (IcedTea 2.6.8) (7u121-2.6.8-2~deb8u1)
# OpenJDK 64-Bit Server VM (build 24.121-b00, mixed mode)

mvn -version

# output
# Apache Maven 3.0.5
# Maven home: /usr/share/maven
# Java version: 1.8.0_121, vendor: Oracle Corporation
# Java home: /usr/lib/jvm/java-8-openjdk-amd64/jre
# Default locale: en, platform encoding: UTF-8
# OS name: "linux", version: "4.9.13-moby", arch: "amd64", family: "unix"

ls -l /usr/lib/jvm/

# output
# lrwxrwxrwx 1 root root   24 May  6  2014 default-java -> java-1.7.0-openjdk-amd64
# lrwxrwxrwx 1 root root   20 Feb  7 20:22 java-1.7.0-openjdk-amd64 -> java-7-openjdk-amd64
# lrwxrwxrwx 1 root root   20 Jan 24 20:58 java-1.8.0-openjdk-amd64 -> java-8-openjdk-amd64
# drwxr-xr-x 5 root root 4096 Apr 26 07:01 java-7-openjdk-amd64
# drwxr-xr-x 7 root root 4096 Apr 25 00:42 java-8-openjdk-amd64
```

I am using docker for mac

```
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Fri Mar 24 00:00:50 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Comment by tianon** [comment]
Doh, definitely something we can and should fix -- see #111 for my initial stab at doing so. :+1:

**Comment by frangarcia** [comment]
Issue fixed, thanks!
